x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/syncjob/ConnectorSyncJobTestUtils.java

```diff
@@ -194,4 +195,8 @@ public static ListConnectorSyncJobsAction.Request getRandomListConnectorSyncJobs
             Collections.singletonList(ConnectorTestUtils.getRandomSyncJobType())
         );
     }
+
+    public static ClaimConnectorSyncJobAction.Request getRandomClaimConnectorSyncJobActionRequest() {
+        return new ClaimConnectorSyncJobAction.Request(randomAlphaOfLength(10), randomAlphaOfLengthBetween(10, 100), Map.of("test", "123"));
```

Comment:
can sync cursor be randomly `null` or this `Map.of("test", "123")`? 

e.g. `randomBoolean() ? Map.of("test", "123") : null`